### PR TITLE
Update description and respect 'Do Not Disturb'

### LINKED ISSUE #54: respect Do-Not-Disturb setting
I love your extension, I just don't know why Gnome doesn't implement this directly. The only problem I have with your extension is that I still get "notified" (blinking) even though I've toggle Do Not Disturb.
Would it be possible to check if this setting is enabled and not do any blinking, and once the setting is turned off, perhaps resume blinking again?

### PR DESCRIPTION
**Pull request by zweif**
The messaging menu is gone and so is the presence switch. This updates the description of the extension and uses the Gnome 'Do Not Disturb' setting instead of the presence status.
Fixes #54

### CODE REVIEW

**Comment by zweif** [comment]
Hey @bellini666, could you take a look at this?

File: src/locale/cs_CZ.po [hunk 1]
```diff
@@ -31,11 +31,17 @@ msgid "Alert background color"
 msgstr "Barva pozadí výstrahy"
 
 #: src/prefs.ui:75 src/prefs.ui:79
-msgid "Alert even if you set notifications to OFF on user menu (default: OFF)"
+#, fuzzy
+msgid "Alert even if you enable 'Do Not Disturb' (default: OFF)"
```
Comment (bellini666) [inline src/locale/cs_CZ.po:35]: I prefer to revert the changes to those translations if they are not going to be updated

File: src/locale/nl_NL.po [hunk 2]
```diff
@@ -32,11 +32,18 @@ msgid "Alert background color"
 msgstr "Meldingskleur"
 
 #: src/prefs.ui:75 src/prefs.ui:79
-msgid "Alert even if you set notifications to OFF on user menu (default: OFF)"
+#, fuzzy
```
Comment (bellini666) [inline src/locale/nl_NL.po:35]: Ditto

File: src/locale/pt_BR.po [hunk 3]
```diff
@@ -30,11 +30,17 @@ msgid "Alert background color"
 msgstr "Cor de fundo do alerta"
 
 #: src/prefs.ui:75 src/prefs.ui:79
-msgid "Alert even if you set notifications to OFF on user menu (default: OFF)"
```
Comment (bellini666) [inline src/locale/pt_BR.po:33]: Ditto

**Review by bellini666 — CHANGES_REQUESTED** [review]
OMG sorry for not seeing this earlier. Somehow I missed the notification for this.

TBH it is being a while since I don't use this extension anymore, so I'm going to trust you that this is working and doing what it is supposed to do :)

Just asked to revert some translations that were not updated and should be good to merge

**Comment by zweif** [comment]
Thanks for looking at this. Do you mean the locale files shouldn't be changed if I cannot translate them properly? 
They were updated with the `make locale-files` command to reflect the changed strings in the code. The `fuzzy` annotation indicates that the annotated string might need some work (which they actually do). This is for example used by Poedit.

I agree that this should be translated properly, but I think leaving these files unchanged instead is not a good solution.
What we could do is
- keep the imperfect translations (as in this PR) and wait for someone to improve them later
- remove the imperfect translations (set empty strings) and wait for someone to add proper ones later
- use some tool like deepl to improve them now - but I cannot really verify the output
- maybe ask previous translators to do it - you could do pt_BR

What do you think?

**Comment by bellini666** [comment]
@zweif Hrmmm I see what you mean. Yeah, I think removing the `fuzzy` and setting those translations to an empty string so the original english string will be used is the best option.

And yes, I can do pt_BR right after this is merged :)

**Comment by zweif** [comment]
@bellini666 I removed the fuzzy strings in cs_CZ.po and nl_NL.po. There were more than expected. Now only pt_BR.po is left :)

**Review by bellini666 — APPROVED** [review]
Ty!

**Comment by bellini666** [comment]
Sent it to review: https://extensions.gnome.org/review/49579

**Pull request merged**